recipes/xmlsec/all/conanfile.py

```diff
@@ -0,0 +1,213 @@
+                "static={}".format(yes_no(not self.options.shared)),
+                "include=\"{}\"".format(";".join(self.deps_cpp_info.include_paths)),
+                "lib=\"{}\"".format(";".join(self.deps_cpp_info.lib_paths)),
+                "with-dl=no",
+                "xslt={}".format(yes_no(self.options.with_xslt)),
+                "iconv={}".format(yes_no(False)),
+                "crypto={}".format(",".join(crypto_engines)),
+            ]
+
+            configure_command = " ".join(args)
+            self.output.info(configure_command)
+            self.run(configure_command)
+
+            # Fix library names
+            def format_libs(package):
+                libs = []
+                for lib in self.deps_cpp_info[package].libs:
+                    libname = lib
+                    if not libname.endswith(".lib"):
+                        libname += ".lib"
+                    libs.append(libname)
+                for lib in self.deps_cpp_info[package].system_libs:
+                    libname = lib
+                    if not libname.endswith(".lib"):
+                        libname += ".lib"
+                    libs.append(libname)
+                return " ".join(libs)
+
+            tools.replace_in_file("Makefile.msvc", "libxml2.lib", format_libs("libxml2"))
+            tools.replace_in_file("Makefile.msvc", "libxml2_a.lib", format_libs("libxml2"))
+            if self.options.with_xslt:
+                tools.replace_in_file("Makefile.msvc", "libxslt.lib", format_libs("libxslt"))
+                tools.replace_in_file("Makefile.msvc", "libxslt_a.lib", format_libs("libxslt"))
+
+            if self.settings.build_type == "Debug":
+                tools.replace_in_file("Makefile.msvc", "libcrypto.lib", "libcryptod.lib")
+
+            self.run("nmake /f Makefile.msvc")
+
+    def _package_msvc(self):
+        with self._msvc_build_environment():
+            self.run("nmake /f Makefile.msvc install")
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+        if not self.options.shared:
+            self._autotools.defines.append("XMLSEC_STATIC")
+        yes_no = lambda v: "yes" if v else "no"
+        configure_args = [
+            "--enable-crypto-dl={}".format(yes_no(False)),
+            "--enable-apps-crypto-dl={}".format(yes_no(False)),
+            "--with-libxslt={}".format(yes_no(self.options.with_xslt)),
+            "--with-openssl={}".format(tools.unix_path(self.deps_cpp_info["openssl"].rootpath) if self.options.with_openssl else "no"),
+            "--enable-mscrypto={}".format(yes_no(False)),   # Built on mingw
+            "--enable-mscng={}".format(yes_no(False)),      # Build on mingw
+            "--enable-docs={}".format(yes_no(False)),
+            "--enable-mans={}".format(yes_no(False)),
```

Comment:
```suggestion
            "--enable-docs=no",
            "--enable-mans=no",
```

Reply:
Why shouldn't we use the lamda function here?

Reply:
When would it ever be a "yes" ?

Reply:
You are right.